packages/rancher-logging/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: oci://ghcr.io/kube-logging/helm-charts/logging-operator:4.8.0
-version: 103.1.2
+version: 103.1.3
```

Comment:
I think this should be an `-rc.1`.

Similar to how Josh's looks here: https://github.com/rancher/charts/pull/4311

Once that's corrected I think this looks good.